docs/fundamentals/toc.yml

```diff
@@ -297,6 +304,17 @@ items:
       - name: 3 - Create a template pack
         href: ../core/tutorials/cli-templates-create-template-pack.md
         displayName: tutorials, cli
+  - name: Integrated Development Environment (IDE)
+    items:
+    - name: Visual Studio
+      displayName: ide,integrated development environment,vs
+      href: https://visualstudio.microsoft.com/vs
```

Comment:
Do we want to link to the docs instead of the product page?

Reply:
Good callout!